backend/corpora/dataset_processing/process.py

```diff
@@ -373,12 +376,20 @@ def process_cxg(local_filename, dataset_id, cellxgene_bucket):
 def main():
     check_env()
     dataset_id = os.environ["DATASET_ID"]
-    local_filename = download_from_dropbox_url(
-        dataset_id,
-        os.environ["DROPBOX_URL"],
-        "local.h5ad",
-    )
-    logger.info("Download complete", flush=True)
+    try:
+        local_filename = download_from_dropbox_url(
+            dataset_id,
+            os.environ["DROPBOX_URL"],
+            "local.h5ad",
+        )
+    except ProcessingCancelled as ex:
+        logging.info(ex.status)
+        sys.exit(0)
+    except ProcessingFailed as ex:
+        logging.error(ex.status)
```

Comment:
What does this do with the actual error and traceback?

Reply:
the traceback and error are in ex.status.upload_message which are populated by the download code.